daily-challenges/count-code/index.test.js

```diff
@@ -1,7 +1,21 @@
-const functionsToTest = require('./index');
+const { countCode } = require("./index")
 
-xdescribe('All tests in this describe will be skipped. Remove the "x" from "describe" to turn them on', () => {
-    test('This test will be skipped', () => {
-        expect(true).toBe(true);
-    });
-});
+describe("Count Code Challenge", () => {})
+describe('Returns the number of times "code" appears in given string, ignoring the "d', () => {
+    test("should return number of times 'co_e' exists in the string", () => {
+        expect(countCode("aaacodebbb")).toBe(1)
+        expect(countCode("cozexxcope")).toBe(2)
+        expect(countCode("codexxcode")).toBe(2)
+    })
+})
+
+xdescribe("Extra credit challenges", () => {
+    test("remove the x and make your function pass these edge cases", () => {
```

Comment:
These tests don't actually match the extra credit from the writeup. Change one or the other so they match. (If you change the writeup, make sure to change the `README.md` file here too)